kbase/narrative
# DATAUP-178 Change button color

# Description of PR purpose/changes

* Changed the Add data button in the data list to blue. 
* Added unit testing of DataList Widget
* Altered attributes of the button to make it more accessible

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-178

# Testing Instructions

To execute new Tests:
* Navigate to local repo for narrative in your terminal
* Execute `make test-frontend-unit`

To see visual color change, open a narrative. View the button when there is and is not data in the dataList.


# Dev Checklist:


# Updating Version and Release Notes (if applicable)
